db/post_migrate/20210922064213_alter_bumped_at_indexes_on_topics.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class AlterBumpedAtIndexesOnTopics < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    execute(<<~SQL)
+    CREATE INDEX CONCURRENTLY idx_public_topics_front_page
+    ON topics (bumped_at,category_id,visible,id)
```

Comment:
There is little point in having anything after `bumped_at`, because `bumped_at` will probably be unique.

Reply:
Good point. I always forget that ordering is important when creating a multiple column index.